ci/win_install_deps.cmd

```diff
@@ -28,7 +28,8 @@ C:\Python39\python.exe -m pip install  numpy==1.19.3 "cython < 3" || goto :error
 C:\Python310\python.exe -m pip install numpy==1.21.3 "cython < 3" || goto :error
 C:\Python311\python.exe -m pip install numpy==1.23.5 "cython < 3" || goto :error
 C:\Python312\python.exe -m pip install numpy==1.26.3 "cython < 3" || goto :error
-C:\Python312\python.exe -m pip install setuptools || goto :error
+:: setuptools 70.2 leads to an error
+C:\Python312\python.exe -m pip install setuptools==70.1.1 || goto :error
```

Comment:
Just adding screenshot here for future reference:

<img width="983" alt="image" src="https://github.com/neuronsimulator/nrn/assets/666852/8dd47731-370b-47d5-926d-677a8b92f67a">